test/pummel/test-timers.js

```diff
@@ -61,7 +61,7 @@ const WINDOW = 200; // Why does this need to be so big?
 
     const t = interval_count * 1000;
 
-    assert.ok(t <= diff && diff < t + (WINDOW * interval_count));
+    assert.ok(Math.abs(diff - t) < WINDOW && diff < t + (WINDOW * interval_count));
```

Comment:
Is the RHS still needed?

Reply:
After some thoughts, I think it is not needed. I may still keep the `* interval_count` for now. Planning to do a follow up PR to reduce the `WINDOW` size as it seems way too big.
```suggestion
    assert.ok(Math.abs(diff - t) < WINDOW * interval_count);
```

Reply:
Thanks for catching this 👍